src/sinks/aws_cloudwatch_logs/request.rs

```diff
@@ -276,6 +344,19 @@ impl Client {
         })
     }
 
+    pub fn describe_group(&self) -> ClientResult<DescribeLogGroupsOutput, DescribeLogGroupsError> {
+        let client = self.client.clone();
+        let group_name = self.group_name.clone();
+        Box::pin(async move {
+            client
+                .describe_log_groups()
+                .limit(1)
+                .log_group_name_prefix(group_name)
```

Comment:
Is this sufficient?

* cloudwatch has log group `x1`, vector looking for `x`, api call returns `x1`
* cloudwatch has log groups `x` AND `x1`, vector looking for `x`, we _HOPE_ it returns `x`, but feels like we're depending on some implementation detail for this.